### ISSUE TITLE
[CAIA A11y Research]: Ask VA Tool (January 2024)

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#66774]
`Pending Invites to CAIA`

### Who's on your team?

```bash
## About your team
Team name: Ask VA (AVA)
OCTO-DE product owner: @clantosswett
Product manager: @ruchishah29
Designer: @t-michaud @beckyphung @tygindraux
FE engineer: None listed
Accessibility specialist: None listed
Content writer: None listed
Product/team Slack channel: #ask-va-public
```

### Do you have an accessibility champion?

No, we don't have a champion (yet)

### Calendar checking

- [ ] Based on this calendar, confirm we have specialists who can support your study in your desired timeframe
- [ ] Confirm you are filing this support ticket at least 1 week in advance of said timeframe, so we have enough heads up to provide support

### Have you reviewed the CAIA capacity calendar?

- [ ] Yes, we have reviewed the CAIA capacity calendar and can confirm specialists are available to support in our research timeframe

### When is the timeframe for your testing?

January 2024

### Supporting artifacts (if any)

- [Product outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/products/Ask-VA-MVP-1-Product-Outline.md)
- [User flows: Health Category User Flow](https://www.sketch.com/s/6a75d0a2-e484-4f1e-8675-ad5747a1a871)
- [Epics/issue that may be helpful](https://github.com/department-of-veterans-affairs/va.gov-team/issues/62304)
-  [Research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/research/Submit%20and%20review%20an%20inquiry/Research%20plan.md)
-  [Conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/research/Submit%20and%20review%20an%20inquiry/Conversation%20guides/Veteran%20guide.md)
...
### Martha's comment in the Epic
[[Epic] CAIA Support for Ask VA team#60696](https://github.com/department-of-veterans-affairs/va.gov-team/issues/60696): I'm opening this request on behalf of the Ask VA (aka AVA) team. The Ask VA team is a new team who will be redesigning the https://ask.va.gov/ tool from scratch. It would be great to assign a CAIA a11y specialist now, in the early stages, so a11y can be considered as early as possible.

### Describe what you need in a few sentences:

An existing product right now that basically serves as a support channel for Veterans to submit a web form, with a question, regarding documents, routed to support groups who receive these inquiries. They are able to respond through Secure Message, in the current portal, with mail an option as well. Inbox and messaging component available.

AVA on the modernized VA.gov platform will be a digital support channel that works with other tools and features on VA.gov and VA ecosystems to quickly and accurately answer submitters' questions. Submitters should feel it is easy and intuitive to submit their issues and receive adequate updates and information that results in a final resolution. Ultimately, we want AVA to feel part of a cohesive and submitter-centered approach to support.

#### Phase 1 

Our Phase 1 will have brought the current Ask VA up to VA.gov design and `accessibility standards`, updated the content to make it more intuitive for our submitters, and sunset the existing front-end portal of AVA.

You can see it in the current dynamics portal here - https://ask.va.gov/

#### Routing determined by:
- Category selection
- There are about 18 categories (think of it as all the business lines of the VA) and then there are sub topics, and data that we collect along with the question to figure out how to get the question answered.
- Topic selection

#### Submission: 
The form can be submitted signed in or not signed in.

```[tasklist]
### Next Steps
- [x] (@sara-amanda) Convert Next Steps to Tasks in Ticket Template
- [x] (@tygindraux ) Post a link to this ticket in the accessibility-help Slack channel and tag Terry Nichols.
- [x] (@coforma-terry ) **Kickoff Meeting** (Terry to schedule with Tyler) for November 2023 [per comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/66774#issuecomment-1770952616)
- [ ] (@tygindraux ) to review the Research Repo label **SFTW: Screenreader** which you can apply to a [search for issues to find studies that included screenreaders](https://github.com/department-of-veterans-affairs/va.gov-research-repository/issues?q=is%3Aissue+label%3A%22SFTW%3A+Screenreader%22), and the issues have links to the artifacts
- [ ] (@tygindraux) Your team's researchers and observers read through platform guidance on research with assistive technology users
- [ ] **CAIA & AKS VA TEAM** - **Calendar Holds**: Place calendar holds for anticipated testing dates (pending feedback from @tygindraux)
- [ ] **CAIA** Specialists are assigned to your research, be sure to add the respective a11y specialist(s) to your session invites (include the Zoom or Teams link).
- [ ] **CAIA** Specialists review your conversation guide and research plan [Research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/research/Submit%20and%20review%20an%20inquiry/Research%20plan.md) and [Conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/research/Submit%20and%20review%20an%20inquiry/Conversation%20guides/Veteran%20guide.md)
- [ ] (@tygindraux) **Optional**: Schedule a Pilot Session to run through your research conversation guide with CAIA A11y Specialists
- [ ] (@tygindraux) **Provide specialists your participant details:** Skill Level of User: ex. Intermediate; Operating System: ex. Windows; Assistive Technology being used: ex. JAWS; Internet Browser: ex. Chrome; and Device Type: Desktop
- [ ] **CAIA** Specialists create a research notes template
- [ ] **CAIA** Post research, specialists will add accessibility findings to your research folder
- [x] **Closing ticket** - confirmed work not needed 4/10/2024
```

**Comment by tygindraux** [comment]
Hi @sara-amanda! Thanks again for opening this ticket.

I'm having a look at the next steps in the description, and I'm wondering when we should kick off this process (is it too soon?). For example, 'Post a link to this ticket in the accessibility-help Slack channel and tag Terry Nichols.' Should we do this now?

Also, so far I've drafted a [Veteran conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/research/Submit%20and%20review%20an%20inquiry/Conversation%20guides/Veteran%20guide.md) and [dependent conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/research/Submit%20and%20review%20an%20inquiry/Conversation%20guides/Dependent%20guide.md) – but I haven't started on our screen reader guide conversation guide yet. Do you have any examples I can reference? If not, I'll draft something up knowing this will be reviewed by y'all. 

Let me know if there's anything else I can be doing at this stage, to make it easier to work with our team. Thanks again!

**Comment by sara-amanda** [comment]
## Original Message
> Hi @sara-amanda! Thanks again for opening this ticket.
> 
> I'm having a look at the next steps in the description, and I'm wondering when we should kick off this process (is it too soon?). For example, 'Post a link to this ticket in the accessibility-help Slack channel and tag Terry Nichols.' Should we do this now?
> 
> Also, so far I've drafted a [Veteran conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/research/Submit%20and%20review%20an%20inquiry/Conversation%20guides/Veteran%20guide.md) and [dependent conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/research/Submit%20and%20review%20an%20inquiry/Conversation%20guides/Dependent%20guide.md) – but I haven't started on our screen reader guide conversation guide yet. Do you have any examples I can reference? If not, I'll draft something up knowing this will be reviewed by y'all.
> 
> Let me know if there's anything else I can be doing at this stage, to make it easier to work with our team. Thanks again!

## Research Resources: Screen Reader Guides

@tygindraux I recall Liz Lanz mentioning something along these lines a few months back in the [DSVA slack channel for inclusive research](https://dsva.slack.com/archives/C033ZS9E7EW/p1689766128291479?thread_ts=1689715634.046869&cid=C033ZS9E7EW). Here is a link to that thread and I can also copy the content and link from that post for you too. I hope this helps! 

> The research repo has a label `SFTW: Screenreader `which you can apply to a [search for issues to find studies that included screenreaders](https://github.com/department-of-veterans-affairs/va.gov-research-repository/issues?q=is%3Aissue+label%3A%22SFTW%3A+Screenreader%22), and the issues have links to the artifacts

## Kickoff Timing
- @tygindraux I think any time is a good time for the kickoff, since it would allow us to plan ahead and be proactive. 
    - I defer to @coforma-terry but I think we could do that this month or next?

**Comment by sara-amanda** [comment]
Updating links: 
-  [Research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/research/Submit%20and%20review%20an%20inquiry/Research%20plan.md)
-  [Conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ask-va/research/Submit%20and%20review%20an%20inquiry/Conversation%20guides/Veteran%20guide.md)

**Comment by sara-amanda** [comment]
Closing out ticket per ok by @tygindraux via DSVA slack on 4/10/2024:

> ... Yes, you're right that it's different! That old ticket (66774) can probably be archived.
